app/scripts/postcss/ums.js

```diff
@@ -0,0 +1,12 @@
+const postcss = require('postcss')
+
+// `um` is a unit representing 16px (`--um` variable). It's temporary. Current `rem` is
+// contextual (depends on screen size). Once we fix that (make `rem` constant) we can
+// replace all ums with rems. Simple as that.
```

Comment:
> Once we fix that (make `rem` constant)

What's preventing us from doing this? Bootstrap?

Reply:
Altho bootstrap may have something to do with that (needs checking), I'm more concerned about the second thing you mentioned – things that will break if we simply force a single value for `rem`.